Bottom: a509b6501b3b226d96eba3429e1f17f784935fcd
Top:    68569a4fcc9a67d8cf78103f37c5c51a28c60f7c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 00:41:32 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/draw.c b/src/draw.c
index d2a828a..1d02376 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -485,7 +485,7 @@ DrawEverything (BoxTypePtr drawn_area)
       SWAP_IDENT = 0;
       DrawMask (drawn_area);
       SWAP_IDENT = save_swap;
-      gui->set_layer (NULL,  SL (FINISHED, 0), 0);
+      gui->set_layer (NULL, SL (FINISHED, 0), 0);
     }
   if (gui->set_layer ("soldermask", SL (MASK, BOTTOM), 0))
     {
